### PR TITLE
haku: grad-clip-norm=0.5 on SOTA stack (Lion stabilization)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Apply gradient clipping with max_norm=0.5 on the SOTA stack. No gradient clipping is currently applied. Modded-NanoGPT and many modern transformers use gradient clipping as a regularizer and training stabilizer. Lion optimizer has sign-based updates that can be sensitive to gradient direction; clipping the pre-sign gradient norm at 0.5 may reduce variance in early training and improve convergence. This is particularly motivated by the tau_y and tau_z channels where we have the largest gaps (×3.27 and ×3.43 vs AB-UPT).

**Single delta from SOTA:** `--grad-clip-norm 0.5` (SOTA has no clipping)

Check available flag: `python train.py --help | grep -i clip`

## Current SOTA (Baseline to beat)

PR #232 (askeladd, model-heads=4), W&B run `r8s2dtnq`
- **Best val abupt:** 9.0650% (ep11)
- **Test abupt:** 10.190%
- Per-axis test: surface_pressure=5.461, wall_shear=9.910, volume_pressure=12.656, tau_x=8.432, tau_y=11.952, tau_z=12.447

## Instructions

First verify the exact flag name:
```bash
cd target/
python train.py --help | grep -i clip
```

Then run:
```bash
cd target/
torchrun --standalone --nproc_per_node=8 train.py \
  --agent haku --optimizer lion --lr 1e-4 --weight-decay 5e-4 \
  --no-compile-model --batch-size 4 --validation-every 1 \
  --train-surface-points 65536 --eval-surface-points 65536 \
  --train-volume-points 65536 --eval-volume-points 65536 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 4 --model-slices 128 \
  --ema-decay 0.999 --lr-warmup-epochs 1 \
  --grad-clip-norm 0.5
```

**Key change:** `--grad-clip-norm 0.5` (SOTA has no gradient clipping)

If the flag name differs from `--grad-clip-norm`, use whatever `--help` shows and report the actual flag used.

## Success Criteria

- Beat SOTA val abupt of 9.0650%
- Report final val abupt and test abupt (from best val checkpoint) with all per-axis metrics

## What to Report

Please report in a PR comment:
1. Exact grad-clip flag used
2. Val abupt at each epoch
3. Best val abupt and the epoch it occurred
4. Test abupt evaluated from the best val checkpoint
5. All per-axis test metrics: surface_pressure, wall_shear, volume_pressure, tau_x, tau_y, tau_z
6. W&B run ID

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #292 because Closing as duplicate of PR #309 (thorfinn/grad-clip-norm-0.5) — both test identical `--grad-clip-norm 0.5` on the SOTA stack. The thorfinn PR will run this experiment. haku will be reassigned to a fresh hypothesis to avoid wasting GPU time on duplicate work..
